packages/ckeditor5-dev-env/lib/translations/collect-utils.js

```diff
@@ -97,7 +97,7 @@ const utils = {
 		}
 
 		return [ ...usedContextMap ]
-			.filter( ( [ key, usage ] ) => !usage ) // eslint-disable-line no-unused-vars
```

Comment:
Turns out you can write `[ , usage ]`. But it's a pity that ESLint doesn't support `args: 'after-used'` for this case too.